code/modules/research/designs/medical_designs.dm

```diff
@@ -301,6 +301,17 @@
 	build_path = /obj/item/organ/internal/cyberimp/mouth/breathing_tube
 	category = list("Medical")
 
+/datum/design/cyberimp_breather/simple
+	name = "Simple Breathing Tube Implant"
```

Comment:
```suggestion
	name = "simple breathing tube implant"
```

Reply:
I don't really want to change it because it will be the odd one out that is just in lowercase.

Reply:
All items should have lower case names.

Reply:
This is for the listing in the protolathe, so it won't have any proper/improper issues.